integrations/gradle-bloop/src/main/scala/bloop/integrations/gradle/model/BloopConverter.scala

```diff
@@ -76,18 +75,16 @@ final class BloopConverter(parameters: BloopParameters) {
         name = getProjectName(project, sourceSet),
         directory = project.getProjectDir.toPath,
         sources = getSources(sourceSet),
-        dependencies = allDependencies.toArray,
+        dependencies = allDependencies.toList,
         classpath = classpath,
         out = project.getBuildDir.toPath,
-        analysisOut = getAnalysisOut(project, sourceSet, targetDir).toPath,
```

Comment:
Not providing this property causes the following error for me with the generated bloop project:
```[E] Attempt to decode value on failed cursor: DownField(analysisOut),DownField(project)```

why did you remove it?